IES Specification Docs/ont.file.ttl

```diff
@@ -0,0 +1,3919 @@
+    sh:class ies:TimeBoundedClass ;
+    sh:severity sh:Warning ;
+    sh:targetSubjectsOf ies:beginBoundOfClass .
+
+ies:beginBoundOfClassRangeShape a sh:NodeShape ;
+    sh:class ies:ParticularPeriod ;
+    sh:severity sh:Warning ;
+    sh:targetObjectsOf ies:beginBoundOfClass .
+
+ies:bookingTypeDomainShape a sh:NodeShape ;
+    sh:class ies:TravelBooking ;
+    sh:severity sh:Warning ;
+    sh:targetSubjectsOf ies:bookingType .
+
+ies:bookingTypeRangeShape a sh:NodeShape ;
+    sh:class ies:ClassOfTravelBooking ;
+    sh:severity sh:Warning ;
+    sh:targetObjectsOf ies:bookingType .
+
+ies:brandingDomainShape a sh:NodeShape ;
+    sh:class ies:PaymentArtefact ;
+    sh:severity sh:Warning ;
+    sh:targetSubjectsOf ies:branding .
+
+ies:brandingRangeShape a sh:NodeShape ;
+    sh:class ies:Organisation ;
+    sh:severity sh:Warning ;
+    sh:targetObjectsOf ies:branding .
+
+ies:cardTypeDomainShape a sh:NodeShape ;
+    sh:class ies:PaymentArtefact ;
+    sh:severity sh:Warning ;
+    sh:targetSubjectsOf ies:cardType .
+
+ies:cardTypeRangeShape a sh:NodeShape ;
+    sh:class ies:ClassOfPaymentArtefact ;
+    sh:severity sh:Warning ;
+    sh:targetObjectsOf ies:cardType .
+
+ies:carrierServiceDomainShape a sh:NodeShape ;
+    sh:class ies:TravelService ;
+    sh:severity sh:Warning ;
+    sh:targetSubjectsOf ies:carrierService .
+
+ies:carrierServiceRangeShape a sh:NodeShape ;
+    sh:class ies:Organisation ;
+    sh:severity sh:Warning ;
+    sh:targetObjectsOf ies:carrierService .
+
+ies:chartersDomainShape a sh:NodeShape ;
+    sh:class ies:ResponsibleActor ;
+    sh:severity sh:Warning ;
+    sh:targetSubjectsOf ies:charters .
+
+ies:chartersRangeShape a sh:NodeShape ;
+    sh:class ies:Transit ;
+    sh:severity sh:Warning ;
+    sh:targetObjectsOf ies:charters .
+
+ies:coercedByDomainShape a sh:NodeShape ;
+    sh:class ies:ResponsibleActorState ;
+    sh:severity sh:Warning ;
+    sh:targetSubjectsOf ies:coercedBy .
+
+ies:coercedByRangeShape a sh:NodeShape ;
+    sh:class ies:ResponsibleActor ;
+    sh:severity sh:Warning ;
+    sh:targetObjectsOf ies:coercedBy .
+
+ies:confidenceDomainShape a sh:NodeShape ;
+    sh:class ies:AssessToBeTrue ;
+    sh:severity sh:Warning ;
+    sh:targetSubjectsOf ies:confidence .
+
+ies:contactDetailsOnBookingDomainShape a sh:NodeShape ;
+    sh:class ies:TravelBooking ;
+    sh:severity sh:Warning ;
+    sh:targetSubjectsOf ies:contactDetailsOnBooking .
+
+ies:cookieOnDeviceDomainShape a sh:NodeShape ;
+    sh:class ies:Cookie ;
+    sh:severity sh:Warning ;
+    sh:targetSubjectsOf ies:cookieOnDevice .
+
+ies:cookieOnDeviceRangeShape a sh:NodeShape ;
+    sh:class ies:Device ;
+    sh:severity sh:Warning ;
+    sh:targetObjectsOf ies:cookieOnDevice .
+
+ies:cookieOriginSiteDomainShape a sh:NodeShape ;
+    sh:class ies:Cookie ;
+    sh:severity sh:Warning ;
+    sh:targetSubjectsOf ies:cookieOriginSite .
+
+ies:cookieOriginSiteRangeShape a sh:NodeShape ;
+    sh:class ies:Webpage ;
```

Comment:
`sh:class` belongs on property shape only, not node shape. Please check

Reply:
sh:class does not have a domain specified in the shacl ontology only a range of rdfs:Class. These shapes could be changed to an sh.node property which points to the NodeShape of the class which in turn would validates that it is an rdfs:Class, but I don't think it is invalid as it is.